peer.go

```diff
@@ -793,6 +797,7 @@ out:
 					Registry:         p.server.invoices,
 					Switch:           p.server.htlcSwitch,
 					FwrdingPolicy:    p.server.cc.routingPolicy,
+					SyncStates:       false,
```

Comment:
Referencing back to my prior commit, this is incorrect as the initial channel reestablishment message should _still_ be sent, even in the case that a channel has just finished its funding process.